config/form_profile_mappings/40-10007.yml

```diff
@@ -0,0 +1,13 @@
+application:
+  applicant:
+    applicantEmail: [contact_information, email]
+    applicantPhoneNumber: [contact_information, us_phone]
+    mailingAddress: [contact_information, address]
+    name: [identity_information, full_name]
+  claimant:
+    address: [contact_information, address]
+    dateOfBirth: [identity_information, date_of_birth]
+    name: [identity_information, full_name]
+    ssn: [identity_information, hyphenated_ssn]
+    email: [contact_information, email]
+    phoneNumber: [contact_information, us_phone]
```

Comment:
@jbalboni does this prefill look right? should i prefill military history?

Reply:
I'll have to think about this a bit. The data structure on the front end is pretty different from the schema, and we have the same issue as on the 686 where we don't know who the person is up front.

Reply:
@lihanli Let's just fill in the `claimant` information for now. So we can remove the `applicant` section in here. And I think if we want to pull in military info or anything else, we'll need to get some UX help.